src/XML/xenc/AbstractEncryptedType.php

```diff
@@ -56,7 +56,7 @@ class AbstractEncryptedType extends AbstractXencElement
      *   The EncryptionMethod object of this EncryptedData. Optional.
      * @param \SimpleSAML\XMLSecurity\XML\ds\KeyInfo|null $keyInfo The KeyInfo object of this EncryptedData. Optional.
      */
-    protected function __construct(
+    public function __construct(
```

Comment:
Should it be abstract?

Reply:
Also, the phpdoc says `EncryptedData constructor`, which is incorrect

Reply:
Why? We have an implementation here... Do you mean maybe the class itself? Because that one is not declared abstract despite its name, and it should be. Changing it 😉